transaction/transaction.go

```diff
@@ -806,3 +807,127 @@ func byte32FromBase64(in string) (out [32]byte, err error) {
 	copy(out[:], slice)
 	return
 }
+
+var emptySchema = types.StateSchema{}
+
+// MakeUnsignedAppCreateTx makes a transaction for creating an application
```

Comment:
Remove `Unsigned` from these function names.

I'm not sure if we should abbreviate to `App` or use `Application`, but we definitely should not use both like we have now.

Reply:
fixed.